addon/components/base/bs-nav/item.js

```diff
@@ -105,7 +105,7 @@ export default Component.extend(ComponentParent, {
       params.push({ queryParams: true, ...query});
     }
 
-    return params.length ? this.get('router').isActive(...params) : this.get('_active');
+    return params.length > 0 ? this.get('router.currentURL') && this.get('router').isActive(...params) : this.get('_active');
```

Comment:
If I got it the bug report at emberjs repo right, we are only consuming `router.currentURL` to ensure that CP is recomputed. This should not change any of the logic here, isn't it? If so, I would move that one to a separate line and add a inline comment. Otherwise it's easily removed by next refactor.

Reply:
done